docs/advanced/app-subgraphs.md

```diff
@@ -1,30 +1,30 @@
 # App Subgraphs
 
-Making a subgraph for an aragon app is not much different from making any other kind of subgraph. To learn how to do that, check out TheGraphs's docs: [https://thegraph.com/docs](https://thegraph.com/docs)
+Making a subgraph for an aragon app is not much different from making any other kind of subgraph. To learn how to do that, check out [TheGraphs's documentation](https://thegraph.com/docs).
 
 In this subgraph template, we merely hook up a bunch of Aragon related data sources that will detect any installed organization, app, token, etc in Aragon deployments. This is hidden from you, so that writting the subgraph is as similar as possible as writting a normal subgraph.
```

Comment:
```suggestion
In our app subgraphs, we merely hook up a bunch of Aragon related data sources that will detect any installed organization, app, token, etc in Aragon deployments. This is hidden from you, so that writing the subgraph is as similar as possible as writing a normal subgraph.
```